testdata/deployment/OCP-10873/signed-service_new.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This file is used to create one SVC, it is better to change file name to svc.json.  Use jq tool to format the json file, 
`
$ cat svc.json | jq .
{
  "kind": "Service",
  "apiVersion": "v1",
  "metadata": {
    "annotations": {
      "service.beta.openshift.io/serving-cert-secret-name": "ssl-key"
    },
    "labels": {
      "name": "hello"
    },
    "name": "hello"
  },
  "spec": {
    "ports": [
      {
        "name": "https",
        "protocol": "TCP",
        "port": 443,
        "targetPort": 8443
      }
    ],
    "selector": {
      "name": "web-server-rc"
    }
  }
}

`